groups/groups.yaml

```diff
@@ -15,113 +11,45 @@ groups:
-      - deads@redhat.com
-      - decarr@redhat.com
-      - dcampau1@gmail.com
-      - divya.mohan0209@gmail.com
-      - eddiezane@gmail.com
-      - ehashman@redhat.com
-      - fabrizio.pandini@gmail.com
-      - fbongiovanni@google.com
-      - fbranczyk@gmail.com
-      - frapposelli@vmware.com
-      - guyjtempleton@googlemail.com
-      - hankang@google.com
-      - hpandey@pivotal.io
-      - hweicdl@gmail.com
-      - ian@coldwater.io
-      - irvi.fa@gmail.com
-      - jameswangel@gmail.com
-      - jayunit100.apache@gmail.com
-      - jbelamaric@google.com
-      - jeef111x@gmail.com
-      - jeremy.r.rickard@gmail.com # SIG Release
-      - jeremyot@google.com
-      - jonathan.berkhahn@gmail.com
-      - jsafrane@redhat.com
-      - jshubheksha@gmail.com
-      - jsturtevant@gmail.com
-      - justin@fathomdb.com
-      - k8s@auggie.dev # SIG Release
-      - kaitlynbarnard10@gmail.com
-      - kbhawkey@gmail.com
-      - kikis.github@gmail.com
-      - kim.andrewsy@gmail.com
-      - klaus1982.cn@gmail.com
-      - katrina.verey@shopify.com # SIG CLI
-      - ksemenov@vmware.com
-      - lachlan.evenson@gmail.com
-      - lauri.d.apple@gmail.com # SIG Release
-      - liggitt@google.com
-      - marosset@microsoft.com
-      - maszulik@redhat.com
-      - matt.farina@gmail.com
-      - michmike@gmail.com
-      - mikedanese@google.com
-      - msau@google.com
-      - mwielgus@google.com
-      - neolit123@gmail.com
-      - nospam.wong@gmail.com
-      - pal.nabarun95@gmail.com #CoCC
-      - patrick.ohly@intel.com
-      - prydonius@gmail.com
-      - quinton@hoole.biz
-      - saadali@google.com
-      - saschagrunert@gmail.com # SIG Release
-      - saveetha13@gmail.com
-      - seans@google.com
-      - shu.mutow@gmail.com
-      - siarkowicz@google.com
-      - szostok.mateusz@gmail.com
-      - tabitha.c.sable@gmail.com
-      - timallclair@gmail.com
-      - tasha.drew@gmail.com
-      - theenjeru@gmail.com
-      - thockin@google.com
-      - tim@scalefactory.com
-      - timothysc@gmail.com
-      - tpepper@vmware.com
-      - vincepri@vmware.com
-      - vince@vincepri.com
-      - vllry@zeitgeistlabs.io
-      - wfender@google.com
-      - wojtekt@google.com
-      - xingyang105@gmail.com
-      - ynli@google.com
-      - jim@nirmata.com
-      - rficcaglia@gmail.com
+      - chizhg@google.com	
+      - chizhg@knative.team
+      - csantanapr@knative.team
+      - devguyio@knative.team
+      - dprotaso@knative.team
+      - dsimansk@knative.team
+      - dsimansk@redhat.com
+      - eng.a.abdalla@gmail.com
+      - fguardia@redhat.com
+      - grantr@knative.team
+      - houshengbo@knative.team
+      - knakayam@redhat.com
+      - lionelvillard@knative.team
+      - matzew@knative.team
+      - mwessend@redhat.com
+      - mwessendorf@gmail.com
+      - n3wscott@knative.team
+      - nak3@knative.team
+      - navidshaikh@knative.team
+      - nshaikh@redhat.com
+      - omerbensaadon@knative.team
```

Comment:
Where is this of emails coming from? Is Omer's email here still valid, in other words could we somehow automate bringing in/validating the users?

Reply:
I see you answered a similar question https://github.com/knative/community/pull/905/commits/96caba7995f8604c0a657695534b715e04e6538a#r799568587